### PR TITLE
B3 migrate performance sms views

### PR DESCRIPTION
**Pull request by biyeun**
@gcapalbo / @orangejenny 
cc @proteusvacuum 

screen shot 2016-05-02 at 8 11 29 pm
screen shot 2016-05-02 at 8 11 16 pm
screen shot 2016-05-02 at 8 11 08 pm

### CODE REVIEW

**Comment by biyeun** [comment]
tested locally

File: corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html [hunk 1]
```diff
@@ -17,13 +17,21 @@
...
-        <form method='post' action="{% url 'performance_sms.delete_performance_messages' domain form.config.get_id %}" >
-            {% csrf_token %}
-            <input type="submit" value="{% trans 'Delete'%}" class="btn btn-danger disable-on-submit">
-        </form>
+        <hr />
+        <div class="alert alert-danger">
+            <form method='post' class="form-horizontal" action="{% url 'performance_sms.delete_performance_messages' domain form.config.get_id %}" >
+                {% csrf_token %}
+                <div class="form-group">
+                    <label class="control-label col-sm-3 col-md-2">{% trans 'Delete Performance Mssage?' %}</label>
```
Comment (czue) [inline corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html]: nit: "Mssage"

File: corehq/apps/performance_sms/views.py [hunk 2]
```diff
@@ -1,57 +1,114 @@
...
 
 
-@reminders_framework_permission
-@toggles.SMS_PERFORMANCE_FEEDBACK.required_decorator()
-def list_performance_configs(request, domain):
-    return render(request, "performance_sms/list_performance_configs.html", {
-        'domain': domain,
-        'performance_configs': dbaccessors.by_domain(domain)
-    })
+class BasePerformanceSMSView(BaseDomainView):
```
Comment (czue) [inline corehq/apps/performance_sms/views.py]: can you remind me why it's important to convert these to class based views? just looking at the diff it seems like the original code is much simpler and easier to understand whereas there's lots of indirection in the CBV. not saying we should switch it back just want to know why we do this.

**Comment by orangejenny** [comment]
Looks fine to me, will give you a chance to respond to Cory's comments.

File: corehq/apps/performance_sms/views.py (hunk 2 above)
Comment (biyeun) [inline corehq/apps/performance_sms/views.py]: template consistency and context setup
decorator works on class method

File: corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html (hunk 1 above)
Comment (biyeun) [inline corehq/apps/performance_sms/templates/performance_sms/edit_performance_config.html]: literally way copy paste. will update

**Comment by gcapalbo** [comment]
looks good to me, will let @czue sign off as he worked on this

**Comment by czue** [comment]
good to merge on my end once that text change is made

**Pull request merged**